README.md

```diff
@@ -1,5 +1,15 @@
 # Actividades de AsturiasHacking
 
+---
+
+<p align="center">
+  <b>
```

Comment:
Aren't the `---` and the line created by the paragraph a little redundant in the result markdown? I'd get rid of one of them

Reply:
Yeah, you are right! 👍